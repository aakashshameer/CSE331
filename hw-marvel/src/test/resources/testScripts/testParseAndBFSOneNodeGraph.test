# test parser and BFS on one node graph

# build graph
LoadGraph oneNodeGraph oneNode.csv

# check graph is built correctly
ListNodes oneNodeGraph
ListChildren oneNodeGraph A

#check BFS
FindPath oneNodeGraph A A