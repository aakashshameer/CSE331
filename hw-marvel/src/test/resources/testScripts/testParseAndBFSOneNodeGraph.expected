# test parser and BFS on one node graph

# build graph
loaded graph oneNodeGraph

# check graph is built correctly
oneNodeGraph contains: A
the children of A in oneNodeGraph are:

#check BFS
path from A to A: